README.md

```diff
@@ -2,14 +2,14 @@
 <a href="https://rtcamp.com/?ref=wp-menu-custom-fields-repo" target="_blank"><img width="200"src="https://rtcamp.com/wp-content/themes/rtcamp-v9/assets/img/site-logo-black.svg"></a>
 </p>
 
-# WP Google Login
-[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+# Log in with Google
+<a href="https://www.repostatus.org/#active"><img src="https://www.repostatus.org/badges/latest/active.svg" alt="Project Status: Active – The project has reached a stable, usable state and is being actively developed."></a>
 
-Minimal plugin which allows WP user to login with google. This plugin can be used as MU plugin too.
+Minimal plugin which allows WordPress users to login using Google. Can also be used as an mu-plugin.
```

Comment:
We are mentioning that the plugin can be used as an mu-plugin then we also need to mention how user/dev can do it under `Setup` instructions. Because in the `mu-plugins` folder user/dev need to load the main plugin file in a separate file under the `mu-plugins` folder. WordPress won't load the files from the folder.

Reply:
The same needs to update in the readme.txt file.

Reply:
Updated